backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/contract/C_Flatrate_Term_StepDef.java

```diff
@@ -344,4 +352,88 @@ private I_C_Flatrate_Term retrieveContract(@NonNull final Map<String, String> ro
 						.create()
 						.firstOnlyNotNull(I_C_Flatrate_Term.class));
 	}
+
+	@When("^extend C_Flatrate_Term identified by (.*) starting from (.*) will fail with message (.*)$")
```

Comment:
I could not find a standard way to declare these, but I would suggest to double quote the actual message. This would help readability I think.
```suggestion
	@When("^extend C_Flatrate_Term identified by (.*) starting from (.*) will fail with message \"(.*)\"$")
```